docs/pages/docs/utilities/types.mdx

```diff
@@ -9,7 +9,9 @@ import { Callout } from "nextra/components";
 
 The `"@/generated"{:ts}` module exports types that are useful for creating reusable helper functions.
 
-## EventNames
+## Indexing functions types
```

Comment:
Subheader for top of file

```
To enable code reuse and maintain type safety for advanced use cases, Ponder offers several utility types.
```

Subheader for indexing function types section (remove "s" from end of "functions")
```
The @/generated module exports utility types that are useful for creating reusable helper functions in your indexing files. Like the `ponder` object, these types are aware of the values you specify in your `ponder.config.ts` and `ponder.schema.ts` files.
```

Subheader for config types section
```
The @ponder/core package exports a utility type for each option passed to `createConfig{:ts}`.
```